examples/filter2/main.go

```diff
@@ -98,6 +99,7 @@ func main() {
 
 	// Send FilterRequest from light node to full node
 	cf := protocol.ContentFilter{
+		PubsubTopic:   relay.DefaultWakuTopic,
```

Comment:
I think we should keep examples to use autosharding API, hence i remember changing it to not use default pubsubTopic.

Reply:
Ignore it, just saw your comment regarding separate issue to track this.